youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -0,0 +1,588 @@
+#: youtube_dl_gui/mainframe.py:236
+msgid "Get command"
+msgstr "קבל פקודה"
+
+#: youtube_dl_gui/mainframe.py:237
+msgid "Open destination"
+msgstr "פתח יעד"
+
+#: youtube_dl_gui/mainframe.py:238
+msgid "Re-enter"
+msgstr "הזן מחדש"
+
+#: youtube_dl_gui/mainframe.py:458
+msgid "Resume"
+msgstr "התחל מחדש"
+
+#: youtube_dl_gui/mainframe.py:480
+msgid "Video"
+msgstr "וידיאו"
+
+#: youtube_dl_gui/mainframe.py:484
+msgid "Audio"
+msgstr "שמע"
+
+#: youtube_dl_gui/mainframe.py:516
+msgid "No items selected. Please pick an action"
+msgstr "לא נבחרו פריטים. נא בחר פעולה"
+
+#: youtube_dl_gui/mainframe.py:516
+msgid "Remove all"
+msgstr "מחק הכל"
+
+#: youtube_dl_gui/mainframe.py:516
+msgid "Remove completed"
+msgstr ""
+
+#: youtube_dl_gui/mainframe.py:534
+msgid "Are you sure you want to remove selected items?"
+msgstr "האם אתה בטוח שברצונך להסיר פריטים נבחרים?"
+
+#: youtube_dl_gui/mainframe.py:546
+msgid "Item is active, cannot remove"
+msgstr "הפריט פעיל, לא ניתן להסיר"
+
+#: youtube_dl_gui/mainframe.py:579
+msgid "Item is not completed"
+msgstr "הפריט לא הושלם"
+
+#: youtube_dl_gui/mainframe.py:668
+msgid "Update in progress. Please wait for the update to complete"
+msgstr "עדכון בתהליך. המתן עד להשלמת העדכון"
+
+#: youtube_dl_gui/mainframe.py:716
+msgid "Logging is disabled"
+msgstr "רישום מושבת"
+
+#: youtube_dl_gui/mainframe.py:891
+msgid "Shutdown"
+msgstr "כיבוי"
+
+#: youtube_dl_gui/mainframe.py:891
+msgid "Shutting down in {0} second(s)"
+msgstr "כיבוי בעוד {0} שניות"
+
+#: youtube_dl_gui/mainframe.py:980
+msgid "No items to download"
+msgstr "אין פריטים להורדה"
+
+#: youtube_dl_gui/mainframe.py:1060
+msgid "Are you sure you want to exit?"
+msgstr "אתה בטוח שאתה רוצה לצאת?"
+
+#: youtube_dl_gui/mainframe.py:1060
+msgid "Exit"
+msgstr "יציאה"
+
+#: youtube_dl_gui/mainframe.py:1301 youtube_dl_gui/mainframe.py:1451
+msgid "Cancel"
+msgstr "בטל"
+
+#: youtube_dl_gui/mainframe.py:1450
+msgid "OK"
+msgstr "אשר"
+
+#: youtube_dl_gui/optionsframe.py:65
+msgid "Reset"
+msgstr "אפס"
+
+#: youtube_dl_gui/optionsframe.py:66
+msgid "Close"
+msgstr "סגור"
+
+#: youtube_dl_gui/optionsframe.py:72
+msgid "General"
+msgstr "כללי"
+
+#: youtube_dl_gui/optionsframe.py:73
+msgid "Formats"
+msgstr "פורמט"
+
+#: youtube_dl_gui/optionsframe.py:74
+msgid "Downloads"
+msgstr "הורדות"
+
+#: youtube_dl_gui/optionsframe.py:75
+msgid "Advanced"
+msgstr "מתקדם"
+
+#: youtube_dl_gui/optionsframe.py:76
+msgid "Extra"
+msgstr "תוספות"
+
+#: youtube_dl_gui/optionsframe.py:315
+msgid "Language"
+msgstr "שפה"
+
+#: youtube_dl_gui/optionsframe.py:318
+msgid "Filename format"
+msgstr "פורמט שם הקובץ"
+
+#: youtube_dl_gui/optionsframe.py:323
+msgid "Filename options"
+msgstr "אפשרויות שם הקובץ"
+
+#: youtube_dl_gui/optionsframe.py:324
+msgid "Restrict filenames to ASCII"
+msgstr "הגבל שמות קבצים ל- ASCII"
+
+#: youtube_dl_gui/optionsframe.py:326
+msgid "More options"
+msgstr "אפשרויות נוספות"
+
+#: youtube_dl_gui/optionsframe.py:327
+msgid "Confirm on exit"
+msgstr "אשר ביציאה"
```

Comment:
This translates to *Which is at the exit*, which again it's probably wrong.

Reply:
https://translate.google.com/#en/iw/Confirm%20on%20exit